lib/_stream_wrap.js

```diff
@@ -4,6 +4,7 @@ const assert = require('assert');
 const util = require('util');
 const Socket = require('net').Socket;
 const JSStream = process.binding('js_stream').JSStream;
+const Buffer = require('buffer').Buffer;
```

Comment:
Isn't `Buffer` available everywhere without requiring it? I was able to apply your changes and run it without the `require()` and it worked fine for me.